lib/fatal_error/fatal_error.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include <arch/cpu.h>
+#include <spm.h>
```

Comment:
Should this be conditional to emphasize that the functionality is only used for certain configurations?

Reply:
Personally, i don't like ifdefing headers, which leads to ifdefing code.